huaweicloud/services/fgs/resource_huaweicloud_fgs_function.go

```diff
@@ -463,6 +475,34 @@ func resourceFgsFunctionV2Read(d *schema.ResourceData, meta interface{}) error {
 	return nil
 }
 
+func updateFunctionTags(client *golangsdk.ServiceClient, d *schema.ResourceData) error {
+	var (
+		oRaw, nRaw  = d.GetChange("tags")
+		oMap        = oRaw.(map[string]interface{})
+		nMap        = nRaw.(map[string]interface{})
+		functionUrn = d.Id()
+	)
+
+	if len(oMap) > 0 {
+		opts := function.TagsActionOpts{
+			Tags: utils.ExpandResourceTags(oMap),
+		}
+		if err := function.CreateResourceTags(client, functionUrn, opts); err != nil {
+			return fmt.Errorf("failed to add tags to FunctionGraph function (%s): %s", functionUrn, err)
```

Comment:
oMap means remove old tags?
Maybe the error message and the function is not correct

Reply:
fixed.